Makefile

```diff
@@ -175,6 +169,19 @@ NO_ZERO_ALLOCATIONS = -DNO_ZERO_ALLOCATIONS=1
 ## end devices with less memory you may want to disable this
 USE_MLOCK = -DUSE_MLOCK=1
 
+## When enabled the default library constructor and destructor
+## will be used to initialize and tear down the library. If
+## you disable this you can call iso_alloc_initialize and
+## iso_alloc_destroy to perform the same functions. Note if
+## you disable this you will want to enable ISO_DTOR_CLEANUP
```

Comment:
Maybe add a check to ensure that if `AUTO_CTOR_DTOR` is disabled, `ISO_DTOR_CLEANUP` is enabled, and error otherwise?

Reply:
Yah it doesn't make a whole lot of sense to have them both disabled.